nxcomp/src/Loop.cpp

```diff
@@ -8844,6 +8844,8 @@ int ParseCommandLineOptions(int argc, const char **argv)
 
 int ParseBindOptions(char **host, int *port)
```

Comment:
This function is completely unused... I'm not sure if we should let the caller handle memory cleanup or do it in here. In general I'd say caller...

Reply:
Hmm, interesting. Should we drop it?

Regarding the memory cleanup: I free the memory just before overwriting the old value. In this special case there isn't an outside call where I could free it...

Reply:
commit dropped